_posts/2018-12-12-dot-css-dot-js-2018.md

```diff
@@ -0,0 +1,292 @@
+
+## 'Reading Hex codes' by [David DeSandro](https://twitter.com/desandro){:target="_blank" rel="noopener noreferrer"}
+
+This was one of the most mind blowing talks of dotCSS by far.
+[David DeSandro](https://twitter.com/desandro){:target="_blank" rel="noopener noreferrer"} begins the talk with saying he has a special ability.
+He can read color hex codes. 
+At that moment the most of us were like what?
+You can tell the color based on the hexcode?
+Is that even possible?
+And well, he proved it to us that you really can tell the color based on the hex code.
+He even showed us how to do so.
+
+And so he told us that it requires 5 steps to read color hex codes.
+1. 3-digit shorthand 
+2. Line graph
+3. Hue from shape 
+4. Lightness from total
+5. Saturation from range
+
+Just like this I bet you have no idea what these steps mean.
+But let us teach you the magic of reading color hex codes as well thanks to David.
+
+To get started he teached us that the best way to describe colors is with the HSL color model.
+HSL stands for Hue, Saturation and Lightness. 
+Hue is the pure pigment of a color and can be described with 12 color names.
+We'll show you the color names later in Step 3.
+Saturation is how vibrant or muted the hue is and can be described as *saturated*, *washed*, *muted* or *gray*.
+Lightness speaks for it's self and describes how light or dark the color is and this can be done with *light*, *middle* or *dark*.
+
+Now you know how you can describe a color all you have to do is to follow the 5 steps.
+And deterimine the Hue, lightness and saturation.
+So let's get started!
+
+### Step 1: 3-digit shorthand
+To better understand the process let's take `#D49B25` as an example.
+The first step is to retrieve the 3-digit shorthand of the hexcode.
+This can be done easily by breaking up the hexcode in 3 pairs (D4 | 9B | 25).
+Keep in mind that each pair represents a value of the RGB color channel.
+Now drop the second number of each pair and that gives us the shorthand code `#D92`.
+
+<img class="image fit" alt="Line graph of shorthand hex" style="max-width: 50%" src="/img/2018-11-08-dot-css-dot-js-2018/shorthand.jpg">
+
+### Step 2: Line graph
+With the shorthand from step one we have to create a linegraph based on the numbers of the hexcode.
+With basic understanding of hexadecimal numbers we can visualize a little line graph for the channel values.
+D is high, 9 is around the middle, 2 is low.
+And that's how we get our litle line graph.
+
+<img class="image fit" alt="Hex linegraph example" style="max-width: 25%;" src="/img/2018-11-08-dot-css-dot-js-2018/hex-linegraph.jpg">
+<img class="image fit" alt="Line graph of shorthand hex for color #D92" style="max-width: 25%;" src="/img/2018-11-08-dot-css-dot-js-2018/linegraph.jpg">
+
+### Step 3: Hue from shape
+This is what we think  it's the most tricky part about reading color hex codes.
```

Comment:
Spatie te veel tussen think en 'it's'. It's best ook veranderen naar 'is' in deze zin, oftewel: "This is in our opinion the most tricky part..."